lib/mail_handler/reply_handler.rb

```diff
@@ -96,24 +96,46 @@ def self.is_oof?(message)
 
     def self.forward_on(raw_message, message = nil)
       forward_to = self.get_forward_to_address(message)
-      IO.popen("/usr/sbin/sendmail -i #{forward_to}", "wb") do |f|
+      IO.popen(%Q(/usr/sbin/sendmail -i "#{forward_to}"), 'wb') do |f|
         f.write(raw_message);
         f.close;
       end
     end
 
     def self.get_forward_to_address(message)
       forward_to = AlaveteliConfiguration.forward_nonbounce_responses_to
+
       if AlaveteliConfiguration.enable_alaveteli_pro
-        pro_contact_email = AlaveteliConfiguration.pro_contact_email
-        original_to = message ? MailHandler.get_all_addresses(message) : []
-        if original_to.include?(pro_contact_email)
+        if addressed_to_both_contacts?(message)
+          forward_to = [
+            forward_to,
+            AlaveteliConfiguration.forward_pro_nonbounce_responses_to
+          ].join(',')
```

Comment:
Would extract `AlaveteliConfiguration.forward_pro_nonbounce_responses_to` into a variable and just `return` within this conditional

```suggestion
          return [forward_to, forward_pro_to].join(',')
```

Reply:
Cleaned up in 4426991